[2025-03-09T06:51:49.892+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 4-Entrenamiento_model.validar_modelo manual__2025-03-09T06:51:40.758941+00:00 [queued]>
[2025-03-09T06:51:49.900+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 4-Entrenamiento_model.validar_modelo manual__2025-03-09T06:51:40.758941+00:00 [queued]>
[2025-03-09T06:51:49.900+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-03-09T06:51:49.913+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): validar_modelo> on 2025-03-09 06:51:40.758941+00:00
[2025-03-09T06:51:49.919+0000] {standard_task_runner.py:57} INFO - Started process 1715 to run task
[2025-03-09T06:51:49.922+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '4-Entrenamiento_model', 'validar_modelo', 'manual__2025-03-09T06:51:40.758941+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/Entrenamiento_mode.py', '--cfg-path', '/tmp/tmpvonz_cmc']
[2025-03-09T06:51:49.925+0000] {standard_task_runner.py:85} INFO - Job 37: Subtask validar_modelo
[2025-03-09T06:51:49.971+0000] {task_command.py:410} INFO - Running <TaskInstance: 4-Entrenamiento_model.validar_modelo manual__2025-03-09T06:51:40.758941+00:00 [running]> on host 2e759f21ee59
[2025-03-09T06:51:50.044+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='4-Entrenamiento_model' AIRFLOW_CTX_TASK_ID='validar_modelo' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T06:51:40.758941+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-09T06:51:40.758941+00:00'
[2025-03-09T06:51:50.095+0000] {python.py:183} INFO - Done. Returned value was: {'mejor_modelo': 'LogReg', 'train_accuracy': 1.0, 'test_accuracy': 1.0, 'todos_resultados': {'KNN': {'train_accuracy': 0.9963636363636363, 'test_accuracy': 0.9855072463768116}, 'LogReg': {'train_accuracy': 1.0, 'test_accuracy': 1.0}, 'LogRegCV': {'train_accuracy': 0.9927272727272727, 'test_accuracy': 1.0}}}
[2025-03-09T06:51:50.114+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=4-Entrenamiento_model, task_id=validar_modelo, execution_date=20250309T065140, start_date=20250309T065149, end_date=20250309T065150
[2025-03-09T06:51:50.136+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-03-09T06:51:50.150+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
